app/typeclasses/confirmation/ConfirmationBreakdownRows.scala

```diff
@@ -49,24 +49,18 @@ object BreakdownRowInstances {
           case Some(activities) if people.isDefined =>
 
             val subQuantity = subscriptionQuantity(subscription)
+
             val registrationFeeRow = submissionRow(subscription)
+            val registrationFeeBreakdownRow = Seq(BreakdownRow(registrationFeeRow.message, subQuantity,
```

Comment:
I would use the indentation style as before, it aids in reading the code to know where seq constituents start and end.